History.md

```diff
@@ -26,6 +26,7 @@ This is the first Express 5.0 alpha release, based off 4.10.1.
     - `res.json(obj, status)` signature - use `res.json(status, obj)`
     - `res.jsonp(obj, status)` signature - use `res.jsonp(status, obj)`
     - `res.send(body, status)` signature - use `res.send(status, body)`
+    - `res.send(status, body)` signature - use `res.status(status).send(body)`
```

Comment:
Yeah, I just noticed that I changed the wrong section of the file. I'll go fix all these, sorry 'bout that.